### ISSUE TITLE
Uncaught TypeError: Cannot read property 'openScopes' of undefined #305

### ISSUE THREAD

**Issue by macabdul9** [atom/bracket-matcher#311]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by Ben3eeE** [comment]
Thanks for reaching out!

We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us half way :grinning:

**Comment by OmgImAlexis** [comment]
[Enter steps to reproduce:]

1. Opened project with `cd ~/code/twistly-frontend && atom .`
2. Error showed up.

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [bracket-matcher](https://github.com/atom/bracket-matcher) package 0.86.0


### Stack Trace

Uncaught TypeError: Cannot read property 'openScopes' of undefined

```
At /Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:55

TypeError: Cannot read property 'openScopes' of undefined
    at TagFinder.module.exports.TagFinder.scopesForPositionMatchRegex (/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:55:28)
    at TagFinder.module.exports.TagFinder.isCursorOnTag (/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:45:25)
    at TagFinder.module.exports.TagFinder.findMatchingTags (/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:174:22)
    at BracketMatcherView.module.exports.BracketMatcherView.updateMatch (/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:128:41)
    at /Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:3:65
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at TokenizedBuffer.module.exports.TokenizedBuffer.markTokenizationComplete (/Applications/Atom.app/Contents/Resources/app/src/tokenized-buffer.js:252:28)
    at TokenizedBuffer.module.exports.TokenizedBuffer.tokenizeNextChunk (/Applications/Atom.app/Contents/Resources/app/src/tokenized-buffer.js:246:27)
    at /Applications/Atom.app/Contents/Resources/app/src/tokenized-buffer.js:206:32
    at /Applications/Atom.app/Contents/Resources/app/node_modules/underscore/underscore.js:666:53
```

### Commands

```
     -0:22.2.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling.seti-compact.seti-icons.seti-animate)
```

### Non-Core Packages

```
autohide-tree-view 0.25.1 
busy-signal 1.4.3 
duplicate-removal 0.1.3 
editorconfig 2.2.2 
file-icons 2.1.11 
git-status 0.3.4 
intentions 1.1.5 
language-docker 1.1.8 
language-mako 0.6.0 
language-nginx 0.8.0 
language-pug 0.0.21 
language-vue 0.23.1 
linter 2.2.0 
linter-pycodestyle 2.1.3 
linter-stylelint 4.0.2 
linter-swagger 0.4.1 
linter-ui-default 1.6.7 
linter-xo 0.23.1 
multi-cursor 2.1.5 
seti-syntax 1.1.3 
seti-ui 1.9.0 
sort-lines 0.14.0 
tree-view-autoresize 2.2.0 
tree-view-git-status 1.4.0 
```

**Comment by Ben3eeE** [comment]
@OmgImAlexis This issue has been fixed in one of the hotfix releases for Atom 1.19.

**Comment by Goo0523** [comment]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.0 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [bracket-matcher](https://github.com/atom/bracket-matcher) package 0.86.0


### Stack Trace

Uncaught TypeError: Cannot read property 'openScopes' of undefined

```
At /Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:55

TypeError: Cannot read property 'openScopes' of undefined
    at TagFinder.module.exports.TagFinder.scopesForPositionMatchRegex (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:55:28)
    at TagFinder.module.exports.TagFinder.isCursorOnTag (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:45:25)
    at TagFinder.module.exports.TagFinder.findMatchingTags (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/tag-finder.js:174:22)
    at BracketMatcherView.module.exports.BracketMatcherView.updateMatch (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:128:41)
    at /Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/bracket-matcher/lib/bracket-matcher-view.js:3:65
    at Function.module.exports.Emitter.simpleDispatch (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at TextBuffer.module.exports.TextBuffer.emitDidChangeTextEvent (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:1552:28)
    at TextBuffer.module.exports.TextBuffer.transact (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/text-buffer/lib/text-buffer.js:924:20)
    at TextEditor.module.exports.TextEditor.transact (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1672:32)
    at /Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1233:30
    at TextEditor.module.exports.TextEditor.mergeSelections (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2616:49)
    at TextEditor.module.exports.TextEditor.mergeIntersectingSelections (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2582:41)
    at TextEditor.module.exports.TextEditor.mutateSelectedText (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1231:25)
    at TextEditor.module.exports.TextEditor.insertText (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:1197:25)
    at TextEditor.object.(anonymous function) [as insertText] (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/node_modules/underscore-plus/lib/underscore-plus.js:77:33)
    at TextEditorComponent.didTextInput (/Users/guosenlin/Applications/Atom.app/Contents/Resources/app/src/text-editor-component.js:1599:28)
```

### Commands

```
     -4:13.8.0 application:open (atom-pane.pane.active)
     -3:19.5.0 markdown-writer:insert-new-line (input.hidden-input)
     -3:19.5.0 editor:newline (input.hidden-input)
  8x -2:59.1.0 core:backspace (input.hidden-input)
  4x -2:21.3.0 core:undo (input.hidden-input)
     -2:17.3.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
  6x -1:16.2.0 core:undo (input.hidden-input)
     -1:12.2.0 core:save (input.hidden-input)
     -0:11.6.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
autocomplete-paths 1.0.5 
emmet 2.4.3 
jquery-snippets 11.0.0 
language-vue 0.23.1 
markdown-writer 2.7.2 
simplified-chinese-menu 5.3.4 
```
